freeipa/src/test/java/com/sequenceiq/freeipa/service/operation/FlowOperationServiceTest.java

```diff
@@ -50,7 +50,7 @@
 
     @BeforeEach
     public void setUp() {
-        underTest = new OperationService();
+        underTest = new FlowOperationService();
         MockitoAnnotations.openMocks(this);
     }
```

Comment:
Why the need for an explicit instantiation & `openMocks()`? `@InjectMocks` takes care of both, so I think the whole `setUp()` can be dropped if no other activities are to be done.